services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/helpers/FlowHelper.groovy

```diff
@@ -57,6 +70,35 @@ class FlowHelper {
                 false, false, "autotest flow", null, null)
     }
 
+    /**
+     * Deletes flow with checking rules on source and destination switches.
+     * It is supposed if rules absent on source and destination switches, the flow is completely deleted.
+     */
+    FlowPayload deleteFlow(String flowId) {
+        def flowEntry = db.getFlow(flowId)
+
+        log.debug("Deleting flow '$flowId'")
+        def response = northbound.deleteFlow(flowId)
+
+        def cookie = [flowEntry.left.cookie, flowEntry.right.cookie]
```

Comment:
Consider renaming to `cookies`, since it is an array and more natural to name it in plural

Reply:
Fixed.